### ISSUE TITLE
Parâmetros de saída - Mínimos Quadrados

### ISSUE THREAD

**Issue by tuliodapper** [diegoeck/ELE405#5]
Pessoal,

dei uma olhada na documentação que está em aprovação do Diego. O parâmetro de saída da função "module_name.ls()" para estimativa dos parâmetros do sistema por Mínimos Quadrados é apenas **theta**. Não seria mais fácil para o usuário receber dois vetores: vetor com coefiecientes **a** e vetor com coeficientes **b**? Deixando eles misturados, o usuário pode se confundir...

Abraço,

Túlio

**Comment by max-feldman** [comment]
Concordo que dessa forma os coeficientes podem ficar um pouco escondidos. Conforme já discutimos, podemos retornar o theta de alguma forma que os coeficientes fiquem "separados" (exemplo: theta.a, theta.b, ...), isso poderá sem feito retornando o theta em forma de classe, dicionário, ou alguma outra.

Acredito que essa documentação possa ser integrada da maneira que está agora, para já existir um arquivo base, e conforme for evoluindo o desenvolvimento dos métodos, vamos atualizando de acordo com a implementação realizada.

**Comment by edumapurunga** [comment]
Aproveitando para voltar ao tópico da normatização da saída das funções.

Deixei as funções saindo [a, b] como estavam.

Quais seriam as nossas opções de estruturas no python para retornar somente uma variável?
